### PR TITLE
DCOS-22014: Add and remove repos with graphql mutations

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
## How to review this

Go to Settings > Package Repositores and add / remove repositories. Should work exactly the same like before.

**Checklist**
- [X] Did you add a JIRA issue in a commit message or as part of the branch name?
- [X] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

**Comment by mesosphere-mergebot-dev** [comment]
This repo has @mesosphere-mergebot integration. You can interact with the following commands.
```
@mesosphere-mergebot ship-it  
@mesosphere-mergebot test  
@mesosphere-mergebot integrate  
```

File: plugins/catalog/src/js/repositories/RepositoriesDelete.js [hunk 1]
```diff
@@ -38,6 +59,7 @@ const deleteRepository$ = deleteEvent$
   })
   .startWith({ pendingRequest: false })
   .catch(error => {
+    console.log(error);
```
[the remark this answers is not in the document]
Comment (Fabs) [reply plugins/catalog/src/js/repositories/RepositoriesDelete.js]: yeah, I was sure I had removed it on add -p

File: plugins/catalog/src/js/repositories/RepositoriesAdd.js [hunk 2]
```diff
@@ -2,13 +2,15 @@
 import React from "react";
 /* eslint-enable no-unused-vars */
 
-import { componentFromStream } from "data-service";
+import { componentFromStream, graphqlObservable } from "data-service";
```
[the remark this answers is not in the document]
Comment (DanielMSchmidt) [reply plugins/catalog/src/js/repositories/RepositoriesAdd.js]: 👍

**Comment by DanielMSchmidt** [comment]
@Fabs I don't know if our work here counts as pairing. If not you can approve in my name :)

**Comment by Fabs** [comment]
@DanielMSchmidt I think I would like to see reviews from two more people.

**Comment by juliangieseke** [comment]
wrong target? 🤔

**Comment by Fabs** [comment]
Yeap, fixing it.

**Comment by Fabs** [comment]
Done. It is now all on master hehe.

**Comment by Fabs** [comment]
I rebased master because of this:
image

**Comment by juliangieseke** [comment]
@Fabs no need to rebase because of tests, Jenkins merges every PR into master before running the job. "auto-rebase" if you like :D

**Pull request merged**